src/cmd/CMakeLists.txt

```diff
@@ -1,3 +1,86 @@
+# Collect source files into the "sources" variable and unit test files into the
+# "gtest_sources" variable.
+gz_get_libsources_and_unittests(sources gtest_sources)
+
+add_library(gz STATIC gz.cc)
+target_link_libraries(gz
+  PUBLIC
+    ${PROJECT_LIBRARY_TARGET_NAME}
+)
```

Comment:
Any reason to create a static library here? We could simply create an executable with
`add_executable(${model_executable} model_main.cc ModelCommandAPI.cc)`

(assuming the old name of ModelCommandAPI.cc (now renamed gz.cc))